react/components/SKUSelector/Wrapper.tsx

```diff
@@ -69,15 +76,21 @@ const getVariationsFromSpecifications = (
       }
     }
   }
+
   return variations
 }
 
-const useVariations = (
-  skuItems: ProductItem[],
-  skuSpecifications: SkuSpecification[],
-  shouldNotShow: boolean,
+const useVariations = ({
+  skuItems,
+  skuSpecifications,
+  shouldNotShow,
+  visibleVariations,
+}: {
+  skuItems: ProductItem[]
+  skuSpecifications: SkuSpecification[]
+  shouldNotShow: boolean
```

Comment:
Just to confirm, these props can't be `undefined`, right?

Reply:
Don't know, but even if they are `undefined` they will always be passed as `undefined` (lol)